src/components/counter.ts

```diff
@@ -7,22 +7,100 @@ import type {Extension} from '../main'
 
 export class Counter {
     private readonly extension: Extension
+    private useDocker: boolean = false
+    private disableCountAfterSave: boolean = false
+    private autoRunEnabled: boolean = false
+    private autoRunInterval: number = 0
+    private commandArgs: string[] = []
+    private commandPath: string = ''
+    private texCountMessage: string = ''
+    private wordCount: string = ''
+    private status: vscode.StatusBarItem
 
     constructor(extension: Extension) {
         this.extension = extension
+        // gotoLine status item has priority 100.5
+        this.status = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Right, 100.6)
```

Comment:
100.6 should be less than 100.5. It would be better the counter is at the right of gotoLine status item. With the current implementation, the counter moves unsteadily.


https://user-images.githubusercontent.com/10665499/149036419-172ecc5d-c667-4fb3-a862-1ba1ab16f6a8.mov

Reply:
You are right. I will move the word count item.